### DISCUSSION TITLE
Mermaid.jsで記述したアーキテクチャ図がSyntax errorになります

### DISCUSSION THREAD

**Discussion by tttol** [increments/qiita-discussions#908] [要望・アイデア・不具合]
## 事象
下記のようなmermaid.jsで記述したアーキテクチャ図をQiitaに記載すると、プレビュー画面でSyntax errorになります。

```
architecture-beta
    group api(cloud)[API]

    service db(database)[Database] in api
    service disk1(disk)[Storage] in api
    service disk2(disk)[Storage] in api
    service server(server)[Server] in api

    db:L -- R:server
    disk1:T -- B:server
    disk2:T -- B:db
```
<img width="1465" alt="スクリーンショット 2024-10-01 8 10 44" src="https://github.com/user-attachments/assets/3a73f1d9-8b0c-431e-b500-765994313f8d">

期待値としては、下図のようにアーキテクチャ図が描画される想定です。
```mermaid
architecture-beta
    group api(cloud)[API]

    service db(database)[Database] in api
    service disk1(disk)[Storage] in api
    service disk2(disk)[Storage] in api
    service server(server)[Server] in api

    db:L -- R:server
    disk1:T -- B:server
    disk2:T -- B:db
```

Mermaid.jsのアーキテクチャ図描画機能はv11.1.0+が必要ですが、添付画像のエラー画面には`Syntax error in text mermaid version 10.9.0`と表示されているため、Qiitaの内部で利用しているMermaid.jsのバージョンが古いことが原因でしょうか？
https://mermaid.js.org/syntax/architecture.html

## 要望
Mermaid.jsのアーキテクチャ図描画機能をQiitaでも利用可能にしていただくことは可能でしょうか？

**Comment by tanaka-asami415** [comment]
@tttol 
ご不便をおかけして申し訳ありません。

ご指摘の通り、バージョンが古いことが原因かと思われます。
時期はお約束できないのですが、バージョンアップの対応も予定しておりますので、お待ちいただけると幸いです🙇

**Reply by P-SiZK** [reply]
@tttol 
お待たせしました！

Mermaid のバージョンを v11.3.0 にアップデートし、アーキテクチャ図が描画できるようになりました！
https://qiita.com/release-notes/2024-10-24

**Reply by tttol** [reply]
@P-SiZK @tanaka-asami415 
ご対応ありがとうございました！
